packages/cli/src/lib/spaces/format.ts

```diff
@@ -0,0 +1,4 @@
+export function displayCIDR(cidr: string[] | undefined) {
+  if (!cidr || cidr.length === 0) return ''
+  return cidr.join(', ')
```

Comment:
This can optionally be simplified:
```suggestion
  return cidr?.join(', ') ?? ''
```